docs/src/openapi-v1.yaml

```diff
@@ -2386,10 +2386,10 @@ components:
           enum:
           - preEnding
           - ending
-        firstSlot:
+        firstPeriod:
```

Comment:
I don't understand this, the variable is just renamed and the type is not changed?

Reply:
Yea so the fields `firstSlot`, and `lastSlot` within the `FundInfo` type has been renamed to `firstPeriod` and `lastPeriod`. I worded the description wrong it should say: 

"updates the fields 'firstSlot' and 'lastSlot' within the type `FundInfo`"

Reply:
Given `FundInfo` is a public type, changing it's member names like this is a breaking change right? So the next release is major?

Reply:
Yes indeed. This would be a breaking change, fixed the commit name.